exercises/practice/pop-count/.gitignore

```diff
@@ -0,0 +1,5 @@
+/node_modules
+/bin/configlet
```

Comment:
Was this file auto-created? It looks ... wrong. These ought to be in the root ignore file.

Reply:
This was an auto-generated file.